packages/xo-web/src/xo-app/pool/tab-advanced.js

```diff
@@ -192,6 +193,23 @@ class PoolMaster extends Component {
   }
 }
 
+@connectStore(() => ({
+  defaultSr: createGetObject((_, { pool }) => pool.default_SR),
+}))
+class SelectDefaultSr extends Component {
+  _onChange = setDefaultSr
+
+  render() {
+    const { defaultSr } = this.props
+
+    return (
+      <XoSelect onChange={this._onChange} value={defaultSr.id} xoType='SR'>
+        <Sr id={defaultSr.id} />
+      </XoSelect>
+    )
+  }
+}
```

Comment:
I meant passing `setDefaultSr` directly as a prop:
```suggestion
class SelectDefaultSr extends Component {
  render() {
    const { defaultSr } = this.props

    return (
      <XoSelect onChange={setDefaultSr} value={defaultSr.id} xoType='SR'>
        <Sr id={defaultSr.id} />
      </XoSelect>
    )
  }
}
```